### ISSUE TITLE
[Bug]: No Output / Output vanishes with Version 2.5.1

### ISSUE THREAD

**Issue by marclangesauels** [Roboroads/laravel-tinker#228]
### Short bug description

Since update to version 2.5.1 my console doesn’t shows any output

### Extended bug description

After updating, my console shows output for a fraction of a second and after that the only output shown is:

```
// Laravel Tinker started at 2022-12-22 10:43:35
*** Execution finished ***
```

I use PhpStorm Version 2022.3 and have the new UI (beta) enabled.

For now i use Version 2.4.0 per manual install which works for me. 

### How to reproduce?

1. Open Tinker Console and dd something

### Expected behavior

Output shown in result window

### Screenshots

![image](https://user-images.githubusercontent.com/67956769/209106144-7f31e3fc-46a2-46ac-826a-03cc8139b08a.png)

### IDE & Version

PhpStorm 2022.3

### PHP Version

7.4

### OS and version

Windows 11

### Laravel Tinker plugin version

2.5.1

### Interpreter type

{"label"=>"Local"}

### Laravel Version

v6.20.44

### Tinker version

v1.0.10

### Psysh version

v0.9.12

**Comment by Algeneralo** [comment]
Same issue here :(

**Comment by Pheasey** [comment]
Same as above. Not pin pointed which of the recent versions this started to occur in.

As a temp fix I've uninstalled the plugin and imported from disk using the most recent release from prior to the recent batches.

https://github.com/Roboroads/laravel-tinker/releases/tag/v2.4.0 release.

**Comment by Roboroads** [comment]
I feel like I know what this caused. Mind if I create a build for y'all to test?

**Comment by marclangesauels** [comment]
No i don´t mind. I will try it if you like @Roboroads

**Comment by SaintPeter** [comment]
Interestingly, I wasn't having any issues until maybe after a reboot.

I'm running 
Windows 10
PHPStorm 2022.3.1
PHP 8.0.15
Laravel Tinker 2.5.1
Laravel Framework 9.42.2

I downgraded to Tinker 2.4.0 and it seems to have resolved the issue for the moment.

**Comment by Roboroads** [comment]
Alright. This might acutally be the same issue as #232 which is fixable.

**Comment by EddyStuc** [comment]
Any update on this?

**Comment by algethamy** [comment]
any update?

**Comment by SaintPeter** [comment]
@algethamy I have been using the beta version he released here:
https://github.com/Roboroads/laravel-tinker/issues/232#issuecomment-1467106592

It works ok - I get double output sometimes, but at least it works.

**Comment by Roboroads** [comment]
I got a RC version, should work on all interpreter types and even dump/dd. Let me know if there's issues <3

[Laravel Tinker-2.6.0-rc1.zip](https://github.com/Roboroads/laravel-tinker/files/11201375/Laravel.Tinker-2.6.0-rc1.zip)

**Comment by Roboroads** [comment]
@SaintPeter Thanks for checking this. However, i'm not able to reproduce this one.

![image](https://user-images.githubusercontent.com/4398301/231205626-f7367e1c-8e43-4633-bf7d-0514dfa058a4.png)

**Comment by Roboroads** [comment]
[Laravel Tinker-2.6.0-rc2.zip](https://github.com/Roboroads/laravel-tinker/files/11201990/Laravel.Tinker-2.6.0-rc2.zip)

@SaintPeter I added <meta charset="UTF-8"> to the HTML of the output.

**Comment by SaintPeter** [comment]
@Roboroads Installed rc2, same output.

I am running PHP 8.0.15, don't know if there is a compatibility issue.

In truth, I don't know much about how unicode is handled or what might cause these characters to be mangled.  It does remind me of what happens to unicode in HTML - I think it get mangled in the same way.

I'll keep using this and let you know if I run into any other issues.

**Comment by Roboroads** [comment]
I'm testing with Laravel 10, with a min. req. for 8.1. In any case, it might be a your-project issue, but it's definately an encoding issue. I'm not sure how the plugin could be responsible since I'm not doing too weird of things.

> I'll keep using this and let you know if I run into any other issues.

Thanks! When there's no feedback before friday I'll release it :)

**Comment by marclangesauels** [comment]
Looks good!
![image](https://user-images.githubusercontent.com/67956769/231368286-afa9902c-7491-4aec-936b-4166a1d259d3.png)

**Comment by Roboroads** [comment]
> Looks good!

Awesome! Thanks for testing as well 🙏

**Comment by dennyweiss** [comment]
The [Laravel Tinker-2.6.0-rc2.zip](https://github.com/Roboroads/laravel-tinker/files/11201990/Laravel.Tinker-2.6.0-rc2.zip) is still not working and the fallback to **v2.4.0 is not valid anymore when using PHPStorm 2023.1.**

### How to reproduce?
When I try to run `echo "abcd";` or  `dd('somthing')`

### Expected behavior



### Context:

OS: MacOS Ventura 13.2.1 (22D68)
Plattform: aarch64
IDE: PHPStorm 2023.1
Docker client: Docker version 23.0.1, build a5ee5b1dfc
Docker Runtime:  colima version 0.5.2
PHP Container: laravel/sail v1.21.2
PHP Version: PHP 8.1.17

**Comment by Roboroads** [comment]
@dennyweiss Unfortunately I do not have an M1/2 macbook. I'm also running the sail container but on linux - without issues.

**Comment by Roboroads** [comment]
Issue fixes in 2.6.0
